src/features/runtime-checks/script-overload.js

```diff
@@ -1,4 +1,66 @@
-import { processAttr } from '../../utils.js'
+import { processAttr, getContextId } from '../../utils.js'
+
+const globalStates = new Set()
+
+function generateUniqueID () {
+    const debug = true
```

Comment:
Do you intend to leave this?

Reply:
Not one really, neither mode is bad tbh. I'll set it to false, getting the debugger state here is a bit of a pain that can wait for another PR.